lib/fs.provider.js

```diff
@@ -149,10 +149,10 @@ class FsProvider extends StorageProvider {
 
   select(query, options, callback) {
     if (callback) callback();
-    const fc = new FsCursor();
-    fc.provider = this;
-    fc.jsql.push({ op: 'select', query, options });
-    return fc;
+    return new FsCursor({
+      provider: this,
+      jsql: [{ op: 'select', query, options }]
+    });
```

Comment:
Se comments above why I don't like this changes

Reply:
As I said, this way is much more better than manually changing internal variables of an object.